src/Writer.php

```diff
@@ -67,6 +68,8 @@ public function export($export, string $writerType): TemporaryFile
             $this->addNewSheet()->export($sheetExport);
         }
 
+        $this->raiseBeforeWritingEvent();
```

Comment:
Same question here

Reply:
Same as above, only for non-queued exports.